doc/source/whatsnew/v0.16.0.txt

```diff
@@ -22,6 +22,10 @@ New features
 
 .. _whatsnew_0160.api:
 
+- The ``read_excel()`` function's sheetname argument now accepts a list and None, to get 
+  multiple or all sheets respectively.  If used, a dictionary is returned.
+- A ``verbose`` argument has been added to ``io.read_excel()``, defaults to False.
```

Comment:
- explain the purpose of ``verbose``.
- add the issue number to this
- add a link to the documentation
- I think add a mini-example of using the new functionaility would be nice here (can be the same as in the docs).